File: g.php
<?php

function calculate($to_val) {
	$NUMBER_ONE = 1;
	$i = $NUMBER_ONE;
	$j = $NUMBER_ONE;
	$tmp = 0;
	for ($val = 0; $val < $to_val; $val++) {
		$tmp = $i + $j;
		$i = $j;
		$j = $tmp;
	}
	return ($j / $i);
}

echo calculate(24) . "\n";

?>

File: g.py
"""
Golden Ratio
"""

def calculate(to_val: int) -> float:
    NUMBER_ONE = int(1)
    i = NUMBER_ONE
    j = NUMBER_ONE
    tmp = 0
    for val in range(to_val):
        tmp = i + j
        i = j
        j = tmp
    return j / i

if __name__ == "__main__":
    print(calculate(24))
